include/dts-functions.sh

```diff
@@ -1067,16 +1070,22 @@ set_intel_regions_update_params() {
         if [ $ME_DISABLED -eq 1 ]; then
           FLASHROM_ADD_OPT_REGIONS+=" -i me"
         else
+          fd_me_locked="yes"
           echo "The firmware binary to be flashed contains Management Engine (ME), but ME is not disabled!"  >> $ERR_LOG_FILE
           print_error "The firmware binary contains Management Engine (ME), but ME is not disabled!"
           force_me_update
         fi
       else
+        fd_me_locked="yes"
         echo "The firmware binary to be flashed contains Management Engine (ME), but ME is not writable!"  >> $ERR_LOG_FILE
         print_error "The firmware binary contains Management Engine (ME), but ME is not writable!"
       fi
     fi
   fi
+  if [ "$fd_me_locked" = "yes" ] && [[ "$SYSTEM_VENDOR" == *"Micro-Star International"* ]]; then
+    print_warning "You can read more about issues with FD or ME on"
```

Comment:
```suggestion
    FLASHROM_ADD_OPT_REGIONS+=" -N"
    print_warning "You can read more about issues with FD or ME on"
```

It will be better to move it here, because whether ME is locked or FD is locked, we must use `-N`. Sometimes ME is not readable and will cause the same error as the hole between ME and BIOS regions.

Reply:
306dd78dbfbb